packages/sdk/src/interfaces/types.ts

```diff
@@ -17,8 +17,13 @@ export enum Chain {
   GOERLI = 5,
   KOVAN = 42,
   HARDHAT_LOCAL = 31337,
+  OPTIMISM = 10,
+  OPTIMISM_KOVAN = 69,
 }
 
+export type L1Chain = Chain.MAINNET | Chain.KOVAN | Chain.HARDHAT_LOCAL
+export type L2Chain = Chain.OPTIMISM | Chain.OPTIMISM_KOVAN
```

Comment:
Is there an analogous l2 id for hardhat?

Reply:
The analogous for L1 hardhat is the ops LE that is spun up by docker compose. It's a transient network used for development, still the l2geth codebase

Reply:
@tynes  Does it have a properties (either static or something I can dynamically read from process.env) such as a network id or any of the other properties you'd see in https://github.com/ethereum-optimism/optimism/pull/2471/files#diff-bf4ab09d50e5349ddc955da5a0346de36b7b4bed9b031fc721d4f18281e742b6R6

Reply:
Chain ID for local L2 network is 420 I think. We should include it here so that it's easy to test frontends against the local deployment. I don't think it has any properties like the ones you listed.

Reply:
Ah cool I'll add an object like I linked but give it my own name and a no-op for etherscan links

Reply:
Seeing 420 is correct